frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,145 @@
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = JASMINE_DEFAULT_TIMEOUT_INTERVAL;
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('displays subscription creation form for selected Operator', async () => {
+    await crudView.isLoaded();
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Operators')));
+    await sidenavView.clickNavLink(['Operators', 'OperatorHub']);
+    await crudView.isLoaded();
+    await catalogView.categoryTabsPresent();
+    await catalogView.categoryTabs.get(0).click();
+    await catalogPageView.catalogTileFor('OpenShift Serverless Operator').click();
+    await browser.wait(until.visibilityOf(installServerlessOperator));
+    await installServerlessOperator.element(by.linkText('Install')).click();
+    await operatorHubView.createSubscriptionFormLoaded();
+    expect(operatorHubView.createSubscriptionFormName.getText()).toEqual(
+      'OpenShift Serverless Operator',
+    );
+    await browser.wait(until.visibilityOf(operatorHubView.createSubscriptionFormInstallMode));
+    await operatorHubView.allNamespacesInstallMode.click();
+
+    expect(operatorHubView.createSubscriptionError.isPresent()).toBe(false);
+    expect(operatorHubView.createSubscriptionFormBtn.getAttribute('disabled')).toEqual(null);
+    await operatorHubView.createSubscriptionFormBtn.click();
+    await crudView.isLoaded();
+    await browser.get(`${appHost}/operatorhub/ns/${testName}`);
+    await crudView.isLoaded();
+    await catalogPageView.clickFilterCheckbox('installState-installed');
+    expect(catalogPageView.catalogTileFor('OpenShift Serverless Operator').isDisplayed()).toBe(
+      true,
+    );
+  });
+
+  it(`displays Operator in "Cluster Service Versions" view for "${testName}" namespace`, async () => {
+    await retry(() => catalogPageView.catalogTileFor('OpenShift Serverless Operator').click());
+    await browser
+      .wait(until.presenceOf(installServerlessOperator), 1000)
+      .then(() => browser.sleep(500));
+    await operatorHubView.viewInstalledOperator();
+    await crudView.isLoaded();
+    await browser.wait(
+      until.visibilityOf(operatorView.rowForOperator('OpenShift Serverless Operator')),
+    );
+    expect(operatorView.rowForOperator('OpenShift Serverless Operator').isDisplayed()).toBe(true);
+  });
+
+  it(`create knative-serving namespace and apply serving.yaml`, async () => {
+    await execSync(`oc new-project knative-serving`);
+    await execSync(
+      `oc apply -f ./packages/dev-console/integration-tests/views/serverless-service.yaml`,
+    );
+    await browser.get(`${appHost}/k8s/cluster/projects/`);
+    await crudView.isLoaded();
+    await browser.wait(until.elementToBeClickable(knativeServingNS));
+    expect(knativeServingNS.isDisplayed()).toBe(true);
+  });
+
+  it('Increase the Machine sets count', async () => {
+    await browser.wait(until.presenceOf(sidenavView.navSectionFor('Compute')));
```

Comment:
It is waiting for compute tab  on admin side but to be sure you are on right perspective do perspective switch to admin and wait for crud view to load.

Reply:
I will update the code accordingly.